### PR TITLE
always bind rails to 0.0.0.0

### PR DESCRIPTION
**Pull request by evgeni**
it defaults to 127.0.0.1 otherwise

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -88,7 +88,7 @@ RUN echo gem '"rdoc"' > bundler.d/container.rb && echo gem '"tzinfo-data"' >> bu
 RUN date -u > BUILD_TIME
 
 # Start the main process.
-CMD bundle exec bin/rails server
+CMD bundle exec bin/rails server -b 0.0.0.0
```
Comment (ohadlevy) [inline Dockerfile:91]: why is this required? I didn't see it was needed before?

**Review by ekohl — APPROVED** [review]
Rails binds to `127.0.0.1` in development and `0.0.0.0` in production. For a docker container I think it makes sense to always bind publicly though.

File: Dockerfile (hunk 1 above)
Comment (evgeni) [reply Dockerfile:91]: we override it in compose:
https://github.com/theforeman/foreman/blob/develop/docker-compose.yml#L24

and without that, you can't publish :3000 on the host, as docker proxy will do the request from 172.16.x.y and that won't be able to reach rails :(

Comment (evgeni) [reply Dockerfile:91]: so yeah, it's probably not needed after https://github.com/theforeman/foreman/pull/7540 as @ekohl points out, but we should be consistent with the compose file (and maybe not override the command there?)

Comment (ekohl) [reply Dockerfile:91]: After #7540, I think this is redundant.

Comment (evgeni) [reply Dockerfile:91]: probably.

@ohadlevy shall we drop the `command` from the compose file? I'd prefer not to have multiple definitions of the same thing, makes tracing errors harder

**Comment by tbrisker** [comment]
what's the decision here @evgeni @ohadlevy ? is this needed at all? i don't mind either way.

**Comment by tbrisker** [comment]
It's been nearly a month with no activity, and latest comment seems to indicate this might not be needed anymore. Closing for now, feel free to reopen if this is still needed.
